src/prommis/roasting/ree_oxalate_roaster.py

```diff
@@ -16,48 +16,119 @@
-model only.
-
-Full material and energy balances are modeled. Note that the standard heats of formation of
-many RE oxalates and RE oxides are unavailable in the literature and, therefore, the average
-values of La, Ce, Pr, and Nd compounds are used in case of unavailable data.
-
-The outlet gas product stream is assumed to have the same temperature as the two solid
-product streams.
-if the product temperature is specified as a user input, the heat duty added to the
-reactor is calculately. If the heat duty is given, the product temperature will be calculated.
-
-Currently, no ports for the solid inlet and outlet streams are used. The mass flow rate
-and composition of the solid reactant are specified as input variables inside the model.
-The mass flow rate and the composition of the product streams are also declared as model
-variables.
+r"""
+IDAES REE Oxalate Roaster Unit Model
+====================================
+
+REE Oxalate Roaster is a unit operation to convert oxalates of rare earth and gangue metal
+elements to metal oxides through thermal decomposition and oxidation. There are 18 rare earth elements
+including `Sc`, `Y`, `La`, `Ce`, `Pr`, `Nd`, `Pm`, `Sm`, `Eu`, `Gd`, `Tb`, `Dy`, `Ho`, `Er`, `Tm`, `Yb`, `Lu`,
+and `Th` in the model. The user can specify a subset of those 18 elements through configuration when creating
+the unit model. The 3 gangue elements considered in the model are `Fe`, `Al`, and `Ca`.
+
+The feed oxalate mixture stream is typically from a precipitator in an upstream process. It is assumed
+that all oxalates are in their hydrate forms. In case the anhydrous oxalate feed flow rates are specified in
+the property package of the solid feed stream, the molar flow rates are converted to the corresponding hydrate flow rates. 
+The molecular formula of an oxalate hydrate can be expressed in a general form as :ce:`RE2(C2O4)3 \cdot xH2O` where RE is
+one of the 18 rare earth elements and x is the number of water molecules associated with the hyrate. The three 
+gangue oxalate hydrates considered in the model are :ce:`Fe2(C2O4)3 \cdot 2H2O`, :ce:`Al2(C2O4)3 \cdot H2O`, and
+:ce:`CaC2O4 \cdot H2O`, for `Fe`, `Al`, and `Ca` elements, respectively.
+
+The feed stream could optionally contain surface moisture. The amount of surface moisture entering the
+reactor is specified by an input variable in the current model rather than a state variable in the property
+package of the solid feed stream.
+
+
+Physical Changes
+----------------
+
+Moisture in the solid feed stream is vaporized.
+
+
+Reactions
+---------
+
+The thermal decomposition reactions modeled for rare earth oxalates and three gaugue oxalates are as
+listed below:
+
+- :ce:`RE2(C2O4)3 \cdot xH2O(s) + 1.5O2(g) -> RE2O3(s) + 6CO2(g) + xH2O(g)`
+- :ce:`Fe2(C2O4)3 \cdot 2H2O(s) + 1.5O2(g) -> Fe2O3(s) + 6CO2(g) + 2H2O(g)`
+- :ce:`Al2(C2O4)3 \cdot H2O(s) + 1.5O2(g) -> Al2O3(s) + 6CO2(g) + H2O(g)`
+- :ce:`CaC2O4 \cdot H2O(s) + 0.5O2(g) -> CaO(s) + 2CO2(g) + H2O(g)`
```

Comment:
cdot is not displaying correctly here (and elsewhere in the documentation)

Reply:
This is also my only suggested change, might need a double slash `\\cdot`.

Reply:
It's still not displaying correctly. I'm not sure of the proper way of doing this w/o using :math:, but I believe you can just google centered dot and paste that symbol instead (·).